codecept

```diff
@@ -25,4 +25,5 @@ $app->add(new Codeception\Command\GenerateGroup('generate:groupobject'));
 $app->add(new Codeception\Command\GeneratePageObject('generate:pageobject'));
 $app->add(new Codeception\Command\GenerateStepObject('generate:stepobject'));
 $app->add(new Codeception\Command\GenerateEnvironment('generate:environment'));
+$app->registerCustomCommands();
```

Comment:
when I was thinking of such feature I found that there is issue with loading time.
You are registering custom commands using current config from `Configuration::config()` but what if `-c` parameter passed and config path is different?